cmake/python-site.py

```diff
@@ -1,11 +1,11 @@
-try:
-    from setuptools.sysconfig import get_python_lib, get_config_vars
-except ImportError:
-    from distutils.sysconfig import get_python_lib, get_config_vars
 import sys
+from sysconfig import get_config_vars, get_paths
 
 if sys.argv[1] == "prefix":
-    print(get_python_lib(True, False, sys.argv[2] if len(sys.argv) > 2 else None))
```

Comment:
There is no exact equivalent of `get_python_lib` in sysconfig. The solution chosen here should work most of the time, but may encounter issues with weird system Python installations. For example, this doesn't work using system Python on macOS. Pip faces the same problem: https://github.com/pypa/pip/blob/22.1/src/pip/_internal/locations/__init__.py#L486